File: PlacementCellBackend/DTOs/PlacementDTOs/PlacementDTO.cs
namespace PlacementCellBackend.DTOs.PlacementDTOs;

public class PlacementDTO
{
    public string StudentId { get; set; } = string.Empty;
    public string StudentName { get; set; } = string.Empty;
    public string CompanyId { get; set; } = string.Empty;
    public string CompanyName { get; set; } = string.Empty;
    public string JobTitle { get; set; } = string.Empty;
    public DateOnly PlacementDate { get; set; }
    public string Package { get; set; } = string.Empty;
}

public class CreatePlacementDTO
{
    public string StudentId { get; set; } = string.Empty;
    public string CompanyId { get; set; } = string.Empty;
    public string JobTitle { get; set; } = string.Empty;
    public DateOnly PlacementDate { get; set; }
    public string Package { get; set; } = string.Empty;
}

File: PlacementicsBackend/src/main/java/com/example/placementicsbackend/dtos/PlacementDTOs/PlacementDTO.java
package com.example.placementicsbackend.dtos.PlacementDTOs;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PlacementDto {

    private String id = "";
    private String studentName = "";
    private String companyId = "";
    private String companyName = "";
    private String jobTitle = "";
    private LocalDate placementDate;
    @JsonProperty("package")
    private String pkg = "";
}
